### PR TITLE
Small addition and fix

### PR DESCRIPTION
**Pull request by hadarhg**
- [x] Open Java SDK API Reference in new tab
- [x] Add external link icon to Java SDK API Reference
- [x] Fix header in WordCount Walkthrough

### CODE REVIEW

**Comment by hadarhg** [comment]
R: @davorbonaci

File: src/_includes/header.html [hunk 1]
```diff
@@ -44,7 +44,10 @@
               <li role="separator" class="divider"></li>
 			  <li class="dropdown-header">SDKs</li>
 			  <li><a href="{{ site.baseurl }}/documentation/sdks/java/">Java SDK</a></li>
-			  <li><a href="{{ site.baseurl }}/documentation/sdks/javadoc/">Java SDK API Reference</a></li>
+			  <li><a href="{{ site.baseurl }}/documentation/sdks/javadoc/" target="_blank">Java SDK API Reference <img src="{{ site.baseurl }}/images/external_link.png"
```
Comment (davorbonaci) [inline src/_includes/header.html]: javadoc/{{ site.latest_release }}

File: src/_includes/header.html [hunk 2]
```diff
@@ -44,7 +44,10 @@
...
+                 width="16" height="16"
+                 alt="External link."></a>
```
Comment (davorbonaci) [inline src/_includes/header.html]: Image source and license?

Comment (hadarhg) [inline src/_includes/header.html]: https://commons.wikimedia.org/wiki/File:External_link_font_awesome.svg
https://creativecommons.org/licenses/by-sa/3.0/deed.en

Comment (davorbonaci) [inline src/_includes/header.html]: Seems like this requires an attribution?

**Comment by davorbonaci** [comment]
LGTM with a new icon from a different source. Merging.

File: src/_includes/header.html (hunk 2 above)
Comment (hadarhg) [inline src/_includes/header.html]: (no need for this, changed to different icon)

**Pull request closed without merging**